package.json

```diff
@@ -30,31 +30,32 @@
     "check-format": "prettier --list-different \"{src,tests}/**/*.ts\""
   },
   "peerDependencies": {
-    "eslint": "^5.0.0 || ^6.0.0 || ^7.0.0"
+    "eslint": "^8.0.0"
```

Comment:
why can't we keep compatibility with other older versions?

Reply:
Well, I tried using our plugin with eslint 5.16.0 and I got some type errors on linting. Not sure how I managed to do that, but I am no longer able to reproduce them. I will revert that change.